test/functional/p2p_filter.py

```diff
@@ -125,8 +153,11 @@ def run_test(self):
 
         self.log.info('Check that we receive a tx in reply to a mempool msg if the filter matches a mempool tx')
         filter_node.merkleblock_received = False
+        filter_node.tx_received = False
```

Comment:
True, I had considered just changing the log message (since it didn't seem entirely correct) but I wasn't really sure what it should be. It looks like @MarcoFalke wrote this test, I'm wondering what he would have in mind?